tests/e2e/logging/test_openshift-logging.py

```diff
@@ -67,6 +73,14 @@ def teardown(cbp_obj, sc_obj):
     """
     The teardown will uninstall the openshift-logging from the cluster
     """
+    cur_dir_files = run_cmd("ls")
+    if 'OperatorSource_Images_Labels.txt' in cur_dir_files:
+        run_cmd(
+            'rm -rf disbale_redhat_operator.yaml OperatorSource_Images_Labels.txt '
+            'OP.yaml token.yaml quay.cluster-logging quay.elasticsearch-operator '
+        )
+    else:
+        logger.error("File Not found")
```

Comment:
We might want to be more specific about which file we were unable to find.

Reply:
I think that the loop should iterate over the file names and does the check as Coady suggested for `os.path.isfile` if so you should use `os.remove` to remove the file.

Reply:
done